src/dom/attributes/foreach/default-foreach-attr.js

```diff
@@ -1,3 +1,69 @@
+/**
+ * ## Display Array and Object Variables: data-f-foreach
+ *
+ * Flow.js provides special handling for model variables that are arrays and objects.
```

Comment:
We use the term "special handling" too many times I think. Maybe just remove this line?